tools/airgap_linter.py

```diff
@@ -178,7 +180,7 @@ def main(argv):
 
     if invalid:
         print(
-            "Airgap check FAILED. This framework will NOT work in an airgap. Fix the detected issues."
+            "Airgap check FAILED. The framework's working code directory contains references to non-https URL resources and/or there are http URL references made inside marathon/config JSON files. Please consider replacing these resources to their HTTPS equivalent. In case of internal Mesos DNS URLs, please mention http links as URI under resources.json. This framework will NOT work in an airgap. Fix the detected issues."
```

Comment:
Wdyt of following rephrasing ?
```suggestion
            "Airgap check FAILED. The framework package definition contains references to non-https URL resources and/or there are invalid URL references in the package definition. Please consider replacing these resources to their HTTPS equivalent. In case of DC/OS internal URLs, please move them to service yaml definitions (which will be populated with FQN on the fly). This framework will NOT work in an air-gapped clusters without fixing the detected issues."
```

Reply:
@takirala I agree with your changes and they make sense now! Thank you. I'm going ahead and committing these changes.